dosbox-staging/dosbox-staging
# Packaging improvements for icons on Linux

# Description
This should be the final PR for getting icon files up to snuff on Linux.

First commit was an oversight when I originally edited this icon Makefile.  The install section was missing the new high-res icons.  This is used by the `create-package.sh` script which creates the package that ends up for download on our website.

Second commit creates a script intended to be run by the end user.  This installs the icons and desktop shortcut in the correct place on their system, either system-wide or user-specific (default).

## Related issues
#3169

# Manual testing
Ran `create-package.sh` script.  Verified the high-res icons plus by new script appears in the created package directory.

Ran my new script `install-icons.sh` and verified the icons get installed.  Verified the `Exec=` line in the .desktop file gets replaced by the real dosbox binary location.  Verified icons get removed with the `--remove` flag.

# Checklist

_Please tick the items as you have addressed them. Don't remove items; leave the ones that are not applicable unchecked._

I have:
